#184948247 Configure ESLint and Git Pre-commit hook

wilbrord2 opened the pull request:
What does this PR do?
This PR setup ESLint and Git Pre-commit hook

Description of Task to be completed?

- installing eslint 
- configure eslint for react 
- setting react rules

#### How should this be manually tested?
`npm format`

What are the relevant pivotal tracker stories?
Story ID: [#184948247](https://www.pivotaltracker.com/story/show/184948247)

wilbrord2 committed:
chore(eslint):Configure ESLint and Git Pre-commit hook to run tests and ESlint format before commits
 -  It'll help check for style violations and flag them.
 - The pre-commit hook will run the following commands
- [Delivers #184948247]

Emile-SHUMBUSHO commented on package.json:
```diff
@@ -7,7 +7,19 @@
     "build": "webpack --mode production",
-    "start": "npm run build && serve -s dist"
+    "start": "npm run build && serve -s dist",
+    "format": "eslint --fix src/**/*.js"
```
I tried to run eslint on my computer and got this kind of error
<img width="1165" alt="Screenshot 2023-04-18 at 18 04 35" src="https://user-images.githubusercontent.com/49757277/232837188-9035f031-9995-484e-a7ff-1b610e2ce820.png">

wilbrord2 committed:
Configure ESLint and Git Pre-commit hook
-setup of eslint
- The pre-commit hook will run the following commands
Delivers#184948247
